src/version.c

```diff
@@ -96,13 +96,12 @@ int get_current_version(char *path_prefix)
 	FILE *file;
 	int v = -1;
 	char *buildstamp;
-	char *src, *dest;
 
-	string_or_die(&buildstamp, "%s/usr/lib/os-release", path_prefix);
```

Comment:
I don't know why we have that version file :-) After considering this some more, I'm inclined to read the version file instead, since it's "formatless" and thus trivial to consume, and we already do this for the conf files under /usr/share/defaults/swupd.

But we should rename the version file to `/usr/share/swupd/version`, or similar. This naming change can be introduced for a format bump, of course. The Clear Linux distro "version" can still live in /usr/lib/os-release, since that's the standard location for people to discover this information, and that file will always be in sync with swupd's version file. I'm also not bothered with the duplicate information, since it is already duplicate.

How does this sound?